datacenter/dtr/2.4/guides/support.md

```diff
@@ -5,12 +5,15 @@ description: Your Docker subscription gives you access to prioritized support. Y
 keywords: support, help
 ---
 
-Your Docker Data Center, or Docker Trusted Registry subscription gives you
-access to prioritized support. The service levels depend on your subscription.
+Your Docker Enterprise Edition subscription gives you access to prioritized
+support. The service levels depend on your subscription.
 
-If you need help, you can file a ticket via:
+Before reaching out to Docker Support, make sure you're listed as an authorized
+support contact for your account. If you're not listed as an authorized
+support contact, find a person that is, and ask them to open a case with
+Docker Support in your behalf.
 
-* [Email](mailto:support@docker.com)
-* [Docker support page](https://support.docker.com/)
-
-Be sure to use your company email when filing tickets.
+You can open a new support case in the [Docker support page](https://support.docker.com/).
+If you're unable to submit a new case in the support page, fill-in the
```

Comment:
"fill in" is two separate words. Why wouldn't they be able to use the support page?

s/email/email address. But this seems like a weird thing to say. Maybe "using an email address associated with the subscription"?

Reply:
This was suggested by the support team. This provides an alternative way for users to open a case if the support page is down, or the designated point of contact cannot be found.